rskj-core/src/main/java/org/ethereum/core/TransactionReceipt.java

```diff
@@ -19,21 +19,17 @@
 
 package org.ethereum.core;
 
-import org.bouncycastle.util.BigIntegers;
-import org.bouncycastle.util.encoders.Hex;
-import org.ethereum.util.RLP;
-import org.ethereum.util.RLPElement;
-import org.ethereum.util.RLPItem;
-import org.ethereum.util.RLPList;
-import org.ethereum.vm.LogInfo;
-
-import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-import static co.rsk.util.ListArrayUtil.nullToEmpty;
-import static org.ethereum.util.ByteUtil.EMPTY_BYTE_ARRAY;
+    import org.bouncycastle.util.BigIntegers;
+    import org.ethereum.util.*;
+    import org.ethereum.vm.LogInfo;
+
+    import java.math.BigInteger;
+    import java.util.ArrayList;
+    import java.util.Arrays;
+    import java.util.List;
+
+    import static co.rsk.util.ListArrayUtil.nullToEmpty;
+    import static org.ethereum.util.ByteUtil.EMPTY_BYTE_ARRAY;
```

Comment:
fix indentation?

Reply:
Not sure I've got the question. This was done automatically by an IDE (`Optimize Imports` command)